test/test_cursor.py

```diff
@@ -1392,6 +1392,14 @@ def test_to_list_empty(self):
         docs = c.to_list()
         self.assertEqual([], docs)
 
+    def test_to_list_length(self):
```

Comment:
Could you create an identical test for CommandCursor?

Reply:
There is a `test_command_cursor_to_list_length`

Reply:
Updated to use aggregate